src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1389,12 +1389,8 @@ internal LookupPathCollection ConstructSearchPatternsFromName(string name, bool
         /// True if the command name is either a provider-qualified or PowerShell drive-qualified
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.11.37', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.166.4/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```